app/setup/routes.py

```diff
@@ -41,6 +43,7 @@ def set_config_data():
     return redirect('/setup/status')
 
 @setup_bp.route('/completed_<int:id>', methods=['POST'])
+@login_required
```

Comment:
Please remove this one aswell.
This endpoint is only used by raspberry devices and must be open